assets/js/navMap.js

```diff
@@ -1631,12 +1631,19 @@ var navMap = (function() {
     },
 
 
-    "filterByTaxon": function(name, preventRefresh, isPhylo) {
+    "filterByTaxon": function(name, preventRefresh) {
```

Comment:
I'm not sure what the parameter `isPhylo` did. Is there a particular reason it was removed here?